docs/aci-interation.md

```diff
@@ -0,0 +1,130 @@
+
+## Compose applications
+
+You can start Compose applications with the `docker compose up` command.
+All containers in the same Compose application are started in the same container group. Service discovery between the containers works using the service name specified in the Compose file.
+Name resolution is achieved by writing service names in an `/etc/hosts` file that is shared automatically by all containers in the container group.
+
+Containers started as part of Compose applications will be displayed along with single containers when using `docker ps`. 
+Their container ID will be of the format: `<COMPOSE-PROJECT>_<SERVICE>`. 
+These containers cannot be stopped, started or removed independently since they are all part of the same ACI container group. You can view each container's logs with `docker logs`.
+You can list deployed Compose applications with `docker compose ls`. This will list only compose applications, not single containers started with `docker run`.
+You can remove a Compose application with `docker compose down`.
+
+## Exposing ports
+
+Single containers and Compose applications can optionally expose ports. For single containers this is done using the `--publish` flag of the `docker run` command and for Compose applications this is done in the Compose file.
+ 
+**Note:** ACI does not allow port mapping (i.e.: changing port number while exposing port), so the source and target ports must be the same when deploying to ACI.
+
+**Note:** All containers in the same Compose application are deployed on the same ACI container group. Several containers in the same Compose application cannot expose the same port when deployed to ACI.
+
+By default, when exposing ports for your application, a random public IP address is associated with the container group supporting the deployed application (single container or Compose application).
+This IP address can be obtained when listing containers with `docker ps` or using `docker inspect`.    
+
+### DNS label name
+
+In addition to exposing ports on a random IP address, you can specify a DNS label name to expose your application on an FQDN of the form: `<NAME>.region.azurecontainer.io`.
+This name can be set with the `--domain` flag when performing a `docker run` or using the `domain` field in the Compose file when performing a `docker compose up`.
+**Note:** The domain of a Compose application can only be set once, if you specify `domain` for several services, the value must be identical.
+
+## Volumes
+
+Single containers and Compose applications can use volumes. In ACI, volumes are implemented as Azure file shares in Azure storage accounts.
+To use a volume when running a single container, specify a storage account and file share name like this: 
+
+```console
+docker run -v <STORAGE_ACCOUNT>/<FILE_SHARE>:/target/path[:ro]
```

Comment:
Super nit: We used a hyphen between words above (i.e.: COMPOSE-PROJECT) but are using an underscore here (i.e.: STORAGE_ACCOUNT).

```suggestion
docker run -v <STORAGE_ACCOUNT>/<FILE_SHARE>:/target/path[:ro]
```

Reply:
fixed